app/lib/services/funding_eligibility.rb

```diff
@@ -133,7 +134,11 @@ def eligible_urns
     end
 
     def ecf_api_funding_lookup
-      @ecf_api_funding_lookup = EcfApi::NpqFunding.with_params(npq_course_identifier: course.identifier).find(trn)
+      @ecf_api_funding_lookup = EcfApi::Npq::PreviousFunding.find_for(
+        trn:,
+        get_an_identity_id:,
+        npq_course_identifier: course.identifier,
+      )
```

Comment:
I am planning on moving all this into the query store and pass the info into this service by just giving it the query store. This is a bigger refactor though so I'll be doing a spike for that separately.